docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -4,6 +4,8 @@ title: "Install Camunda 8 on an EKS cluster"
 description: "Set up the Camunda 8 environment with Helm and an optional Ingress setup on Amazon EKS."
 ---
 
+<!-- (!) Note: Please ensure that this guide maintains a consistent structure and presentation style throughout, as with docs/self-managed/setup/deploy/openshift/terraform-setup.md. The user should have a similar experience when reading both guides. -->
```

Comment:
I wanted to know if we could merge this PR or if we were waiting for an alpha or minor. I didn't see any labels, but I did see this comment was still here.  I'll let the assignee or reviewers decide when to merge.

Reply:
My bad, didn't have enough time to review some related changes in other repos.
Running through those as we speak and Léo is back on Friday and would assume him to do the last adjustments then + merge.

The changes would just be references to different branches as those are currently on a feature branch.